### PR TITLE
Motionlayout

### PR DESCRIPTION
**Pull request by tolia-ogurt**

### CODE REVIEW

File: app/src/main/java/com/example/trempel/BindingAdapters.kt [hunk 1]
```diff
@@ -16,8 +16,8 @@ fun loadImage(view: ImageView, url: String?) {
 }
 
 @BindingAdapter("android:rating")
-fun setRating(view: RatingBar, rate: Float) {
-    view.rating = rate
+fun setRating(view: RatingBar, rate: Double) {
+    view.rating.compareTo(rate)
```
Comment (DaryaErofeeva) [inline app/src/main/java/com/example/trempel/BindingAdapters.kt]: Looks redundant, please remove

File: app/src/main/res/xml/pdp_fragment_scene.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MotionScene xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <ConstraintSet android:id="@+id/start">
+        <Constraint
+            android:id="@+id/iv_product"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/height_iV_price"
```
Comment (DaryaErofeeva) [inline app/src/main/res/xml/pdp_fragment_scene.xml]: Please, rename dimen to `iv_product_height`

File: app/src/main/res/xml/pdp_fragment_scene.xml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            app:layout_constraintVertical_bias="0.0" />
+    </ConstraintSet>
+
+    <ConstraintSet android:id="@+id/end">
+        <Constraint
+            android:id="@id/iv_product"
+            android:visibility="gone" />
+        <Constraint
+            android:id="@+id/view"
+            android:layout_marginTop="56dp"
```
Comment (DaryaErofeeva) [inline app/src/main/res/xml/pdp_fragment_scene.xml]: Please, extract `56dp` to dimens file

File: app/src/main/java/com/example/trempel/BindingAdapters.kt (hunk 1 above)
Comment (tolia-ogurt) [reply app/src/main/java/com/example/trempel/BindingAdapters.kt]: done

File: app/src/main/res/xml/pdp_fragment_scene.xml (hunk 2 above)
Comment (tolia-ogurt) [reply app/src/main/res/xml/pdp_fragment_scene.xml]: done

File: app/src/main/res/xml/pdp_fragment_scene.xml (hunk 3 above)
Comment (tolia-ogurt) [reply app/src/main/res/xml/pdp_fragment_scene.xml]: done

**Review by DaryaErofeeva — APPROVED** [review]

**Pull request merged**